src/main/java/org/checkerframework/checker/androidresource/AndroidResourceAnnotatedTypeFactory.java

```diff
@@ -5,6 +5,7 @@
 import org.checkerframework.checker.androidresource.qual.ResourceTop;
 import org.checkerframework.checker.androidresource.qual.container.*;
 import org.checkerframework.checker.androidresource.qual.res.*;
+import org.checkerframework.com.google.common.collect.ImmutableMap;

```

Comment:
I think in general we have not been using these Guava dependencies and it just happens to be available because of dependencies from AFU pulling them in.
Would it be much longer if you used just standard Java APIs?

Reply:
Fixed.